src/tile/tile.ts

```diff
@@ -0,0 +1,32 @@
+import chalk from "chalk";
+
+enum Colors {
+    'yellow',
```

Comment:
You do not need the single quotes here :)

You should also add values to you enums e.g.

```typescript
enum Colors {
    Yellow = 0,
    // ...
}
```

Reply:
Enums are discouraged in typescript. https://www.executeprogram.com/blog/typescript-features-to-avoid

Reply:
Done